### ISSUE TITLE
Instance segmentation 

### ISSUE THREAD

**Issue by thangngoc89** [yifangao112/DeSAM#1]
Thank you for releasing your works. May I ask if this model is suitable for instance segmentation? Like there are multiple masks of the same object type.

TIA

**Comment by sliver2000** [comment]
Hi @thangngoc89,

Yes, DeSAM is suitable for instance segmentation tasks. It can do the same thing as U-Net, though it does not currently support multi-class segmentation. Multi-class segmentation requires a slight modification of the IoU prediction head.

**Comment by yifangao112** [comment]
Hi @thangngoc89,

Yes, DeSAM is suitable for instance segmentation tasks. It can do the same thing as U-Net, though it does not currently support multi-class segmentation. Multi-class segmentation requires a slight modification of the IoU prediction head.

**Comment by thangngoc89** [comment]
@yifangao112 thank you for answering. I actually only need a single class instance segmentation. How would I prepare data for such case? 

TIA

**Comment by yifangao112** [comment]
Hi @thangngoc89,

The format of our raw data follows nnU-Net. you can find the code to convert the dataset (for both biomedical and non-biomedical images) at this [link](https://github.com/MIC-DKFZ/nnUNet/blob/nnunetv1/documentation/dataset_conversion.md). In addition, it is necessary to modify the [code](https://github.com/yifangao112/DeSAM/blob/main/utils/utils.py/#L56) which divides the data into training set, validation set and test set.

**Comment by thangngoc89** [comment]
@yifangao112 thank you very much
